go_api/infrastructure/winner.go

```diff
@@ -61,3 +61,23 @@ func (w *WinnerInfrastructure) Delete(id int) error {
 	w.db.Debug().Delete(&winner, id)
 	return nil
 }
+
+// ユーザーに紐づいた全ウィナーの取得
+func (w *WinnerInfrastructure) FindAllLinkUser() (*domain.WinnersIncludeUsers, error) {
```

Comment:
```suggestion
func (w *WinnerInfrastructure) FindAllLinkUser() (*domain.Users, error) {
```
自分が最初`WinnersIncludeUsers`構造体を作りましたが、これってつまり`Users`の構造体なのでこれでいいと思います！